routes/api.mjs

```diff
@@ -48,4 +49,33 @@ router.post("/db/add/", async (req, res) => {
   });
 });
 
+//Mayank Tamakuwala's Part starts here
+router.post("/db/addReview/", async (req, res) => {
+  const insertedId = await insertReview({ data: req.body.data });
+  res.status(200).send({
+    message: insertedId,
+  });
+});
+
+router.post("/popularPicks/", async (req, res) => {
+  try {
+    const requestCookies = req.cookies;
+    if (requestCookies["uev2.loc"] === undefined) {
```

Comment:
We are now using this check in a few locations, so let's turn it into a function instead that we can reuse. Then lets replace the check in the other calls as well (where needed). We can call it something like `doesLocationCookieExist(cookies)` and respond with a 400 error message if the cookie we are looking for does not exist.